Bottom: 16b3b7089b0e7cd2de750935d14a814adf1d1d61
Top:    5cf3cb21f6f98911f3104c95fda4dd2c43ba8fa3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 22:09:13 +0000

Refresh of debug-polygon-stuff

---

diff --git a/src/polygon1.c b/src/polygon1.c
index f944f9f..74cb0cf 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -50,6 +50,8 @@
 #include "rtree.h"
 #include "heap.h"
 
+#include "pcb-printf.h"
+
 #define ROUND(a) (long)((a) > 0 ? ((a) + 0.5) : ((a) - 0.5))
 
 #define EPSILON (1E-8)
